### PR TITLE
CF2018 Language Updates

### PR DESCRIPTION
**Pull request by srbaxter**
#1209

### CODE REVIEW

File: data/en/arraygetmetadata.json [hunk 1]
```diff
@@ -1,30 +1,37 @@
 {
-	"name":"arrayGetMetadata",
+	"name":"getMetadata",
```
Comment (pfreitag) [inline data/en/arraygetmetadata.json]: @srbaxter looks like you are overwriting the docs for `arrayGetMetadata` with docs for `getMetadata` 

`arrayGetMetadata` is a valid function, I think you had meant to modify `getmetadata.json` instead? 

Changes to the other files look good.

Comment (srbaxter) [reply data/en/arraygetmetadata.json]: Whoops, yes, sorry

**Comment by srbaxter** [comment]
Updated

**Comment by pfreitag** [comment]
Wonderful, thanks @srbaxter

**Pull request merged**